src/server/utilities/cspHeader/index.js

```diff
@@ -74,6 +74,7 @@ const advertisingDirectives = {
     'https://tpc.googlesyndication.com',
     'https://*.safeframe.googlesyndication.com',
   ],
+  styleSrc: ['https://fonts.googleapis.com/css?family=*'],
```

Comment:
I not entirely sure wildcards work/are valid at the point in the url? I _think_ they are only valid after the protocol (`https://`). Also, should this be in a `fontSrc` section; is to download a font file or some css?

Reply:
Hello @andrewscfc, 

In regards to `styleSrc` I have simplified the rule to only include the domain name, as I _believe_ that is all that is required to load style sheet. 

As to your question, the `styleSrc` rule is for downloading CSS. That said the CSS itself contains a colleciton of `@font-face` statements which download fonts from a seperate domain. So I have also added a `fontSrc` rule to cater for it.

Would you mind letting me know your thoughts?

Reply:
I think if the `font-face` declarations are coming from the domain you've added to `fontSrc` then this PR should be ready to go?

Reply:
Hi @andrewscfc,

I belive they so, you see the domain in question in the [CSS file mentioned above](https://fonts.googleapis.com/css?family=Archivo|Arimo|Bitter|EB+Garamond|Lato|Libre+Baskerville|Libre+Franklin|Lora|Google+Sans:regular,medium|Material+Icons|Merriweather|Montserrat|Mukta|Muli|Nunito|Open+Sans:400,600,700|Open+Sans+Condensed:300,400,600,700|Oswald|Playfair+Display|Poppins|Raleway|Roboto|Roboto+Condensed|Roboto+Slab|Slabo+27px|Source+Sans+Pro|Ubuntu|Volkhov). Believe that's all that's required to enable google fonts in a CSP environment, the domain of the originating CSS, and the domain for the font server.

Are you able to approve this PR? And how do I go about getting a second approval so that we can test this code?

Reply:
Thanks @jose-de-sousa I'll ask around for another approval now, ✅ from me

Reply:
Thanks @andrewscfc, 👍